specs/spark-application.yaml

```
@@ -0,0 +1,38 @@
+apiVersion: "sparkoperator.k8s.io/v1beta1"
+kind: SparkApplication
+metadata:
+  name: mock-task-runner
+spec:
+  type: Scala
+  mode: cluster
+  image: gcr.io/spark-operator/spark:v2.4.4-gcs-prometheus
+  imagePullPolicy: Always
+  mainClass: MockTaskRunner
+  mainApplicationFile: "https://infinity-artifacts.s3.amazonaws.com/scale-tests/dcos-spark-scala-tests-assembly-2.4.0-20190325.jar"
+  arguments:
+    - "5"
+    - "120"
+  sparkConf:
+    "spark.scheduler.maxRegisteredResourcesWaitingTime": "2400s"
+    "spark.scheduler.minRegisteredResourcesRatio": "1.0"
+  sparkVersion: "2.4.0"
```

Upgrade Spark version in order to comply with image version (like [here](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/examples/spark-pi-prometheus.yaml))
```suggestion
  sparkVersion: "2.4.4"
```